### PR TITLE
Frontend: com_finder view review

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Frontend: com_finder view review

### Testing Instructions

Test the com_finder search in the frontend

### Expected result

Works

### Actual result

Works, with cs issues

### Documentation Changes Required

none

### CODE REVIEW

File: components/com_finder/views/search/tmpl/default_form.php [hunk 1]
```diff
@@ -63,30 +63,29 @@
 
 <form id="finder-search" action="<?php echo JRoute::_($this->query->toUri()); ?>" method="get" class="form-inline">
 	<?php echo $this->getFields(); ?>
-
-	<?php
-	/*
-	 * DISABLED UNTIL WEIRD VALUES CAN BE TRACKED DOWN.
-	 */
```
Comment (brianteeman) [inline components/com_finder/views/search/tmpl/default_form.php:70]: should this be removed? It seems to be beyond the scope of the PR

Comment (zero-24) [reply components/com_finder/views/search/tmpl/default_form.php:70]: I have just reverted that one. To be removed in a different PR.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c5c2d8c1beb95623155c674f3ec13a09a82316e0<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17232">issues.joomla.org/tracker/joomla-cms/17232</a>.</sub>

**Comment by sandewt** [comment]
1) Smart Search is NOT indexed:
If I entered par example 'joomla' as a search term, there is the following message:
Warning: Invalid argument supplied for foreach() in ..\components\com_finder\views\search\tmpl\default_results.php on line 49. (NOK)

2) After indexing, there is NO warning message. (OK)

* Joomla! 3.8.0-beta3-dev Development<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17232">issues.joomla.org/tracker/joomla-cms/17232</a>.</sub>

**Comment by zero-24** [comment]
> Warning: Invalid argument supplied for foreach() in ..\components\com_finder\views\search\tmpl\default_results.php on line 49. 

@sandewt hmm. i have not changed that thing. Can you please double check that the same issue exists in staging and open a new issue where you describe how to reproduce that issue? Thanks.

**Comment by sandewt** [comment]
I opened a new issue #17731. 
After indexing, there is also a warning message if the search term is unkown (see above).
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17232">issues.joomla.org/tracker/joomla-cms/17232</a>.</sub>

**Comment by n9iels** [comment]
I have tested this item :white_check_mark: successfully on b45941c79ef8cb1048455d53ac8311a3c55e25a4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17232">issues.joomla.org/tracker/joomla-cms/17232</a>.</sub>

**Comment by ghost** [comment]
@Quy can you please retest?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b45941c79ef8cb1048455d53ac8311a3c55e25a4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17232">issues.joomla.org/tracker/joomla-cms/17232</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Pull request merged**